packages/core/mesh/teleport/src/muxing/muxer.ts

```diff
@@ -175,12 +196,22 @@ export class Muxer {
     }
     this._destroying = true;
 
-    this._sendCommand({
-      destroy: {
-        error: err?.message,
-      },
-    });
-    this._dispose();
+    Promise.resolve(
```

Comment:
convert to try/catch/finally?

Reply:
i'm having some troubles making it work with `@synchronized` tag on `teleport.destroy`, so i'll squeeze those change to the next iteration of network updates